packages/client-api-schema/src/generated-schema.json

```diff
@@ -592,6 +592,29 @@
       ],
       "type": "object"
     },
+    "DomainBudget": {
```

Comment:
Did you mean to check this in?

Reply:
I ran yarn at the root of the monorepo, and I think this caused the change?